jwebunit-webdriver-plugin/pom.xml

```diff
@@ -9,17 +29,7 @@
     <artifactId>jwebunit-webdriver-plugin</artifactId>
     <name>Webdriver Plugin</name>
     <description>Webdriver plugin for JWebUnit.</description>
-    <build>
-        <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-surefire-plugin</artifactId>
-                <configuration>
-                    <testFailureIgnore>true</testFailureIgnore>
```

Comment:
Webdriver implementation doesn't cover 100% of tests so you should probably keep that.